### PR TITLE
add amlarc examples

### PR DESCRIPTION
**Pull request by JKrame**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

- Added examples for AMLARC deployments

fixes #

### CODE REVIEW

**Comment by lostmygithubaccount** [comment]
please add a `.sh` script in the `cli` directory which tests these endpoint(s) similar to the others

**Review by Chris113113 — APPROVED** [review]

File: cli/endpoints/amlarc/managed/canary-declarative-flow/1-create-endpoint-with-blue.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+$schema: https://azuremlsdk2.blob.core.windows.net/latest/onlineEndpoint.schema.json
+name: my-endpoint
+type: online
+auth_mode: aml_token
+traffic:
+  blue: 100
+identity:
+  type: system_assigned
+target: azureml:/subscriptions/{{subscriptionId}}/resourceGroups/{{resourceGroup}}/providers/Microsoft.MachineLearningServices/workspaces/{{workspaceName}}/computes/{{computeName}}
```
Comment (snowei) [inline cli/endpoints/amlarc/managed/canary-declarative-flow/1-create-endpoint-with-blue.yaml:9]: I prefer the short format.
azureml:{{computeName}}

File: cli/endpoints/amlarc/managed/canary-declarative-flow/1-create-endpoint-with-blue.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      code: 
+        local_path: ../../model-1/onlinescoring/
+      scoring_script: score.py
+    environment:
+      name: env-model1
+      version: 1      
+      path: .
+      conda_file: file:../../model-1/environment/conda.yml
+      docker:
+        image: mcr.microsoft.com/azureml/intelmpi2018.3-ubuntu16.04:20210301.v1
```
Comment (snowei) [inline cli/endpoints/amlarc/managed/canary-declarative-flow/1-create-endpoint-with-blue.yaml:34]: This needs to be updated later when the new base image is released.
This base image doesn't support swagger generation.

File: cli/endpoints/amlarc/managed/canary-declarative-flow/1-create-endpoint-with-blue.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+$schema: https://azuremlsdk2.blob.core.windows.net/latest/onlineEndpoint.schema.json
+name: my-endpoint
```
Comment (snowei) [inline cli/endpoints/amlarc/managed/canary-declarative-flow/1-create-endpoint-with-blue.yaml:2]: This's not a must and it won't work as we specify the endpoint name in az cli command.
I prefer deleting name setting.

File: cli/endpoints/amlarc/managed/canary-declarative-flow/2-scale-blue.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    version: 1
+  name: blue
+  scale_settings:
+    min_instances: 2
+    scale_type: auto
+identity:
+  type: system_assigned
+name: my-endpoint
+target: azureml:/subscriptions/589c7ae9-223e-45e3-a191-98433e0821a9/resourceGroups/localtest-r6-amlarc-rg/providers/Microsoft.MachineLearningServices/workspaces/localtest-r6-amlarc-ws/computes/localtest-r6-aml
+traffic:
```
Comment (snowei) [inline cli/endpoints/amlarc/managed/canary-declarative-flow/2-scale-blue.yaml:25]: How about keep the same order of all settings?

**Review by snowei — CHANGES_REQUESTED** [review]

File: cli/endpoints/amlarc/managed/canary-declarative-flow/1-create-endpoint-with-blue.yaml (hunk 3 above)
Comment (JKrame) [reply cli/endpoints/amlarc/managed/canary-declarative-flow/1-create-endpoint-with-blue.yaml]: name setting has been removed

File: cli/endpoints/amlarc/managed/canary-declarative-flow/2-scale-blue.yaml (hunk 4 above)
Comment (JKrame) [reply cli/endpoints/amlarc/managed/canary-declarative-flow/2-scale-blue.yaml]: Yamls have been reordered

File: cli/endpoints/amlarc/managed/canary-declarative-flow/1-create-endpoint-with-blue.yaml (hunk 1 above)
Comment (JKrame) [reply cli/endpoints/amlarc/managed/canary-declarative-flow/1-create-endpoint-with-blue.yaml]: changed to {{fullyQualifiedComputeId}} for clarity for our customers

File: cli/endpoints/amlarc/managed/canary-declarative-flow/1-create-endpoint-with-blue.yaml (hunk 2 above)
Comment (JKrame) [reply cli/endpoints/amlarc/managed/canary-declarative-flow/1-create-endpoint-with-blue.yaml:33]: Will update this when new image is released

**Comment by krabhishek8260** [comment]
@JKrame why are the failed inside amlarc/managed?

**Comment by JKrame** [comment]
@krabhishek8260 it just mirrors the folder structure for MIR

**Comment by krabhishek8260** [comment]
LGTM

**Review by lostmygithubaccount — CHANGES_REQUESTED** [review]
please add a `.sh` script in the `cli` directory which tests these endpoint(s) similar to the others - there is currently nothing testing these examples

**Pull request merged**